webassembly/app.html

```diff
@@ -203,7 +213,7 @@ <h1 class="title">F3D Web</h1>
           });
 
         // load the file located in the virtual filesystem
-        openFile('f3d.vtp')
+        openFile('f3d.vtp');
```

Comment:
what does `;` means in js ?

Reply:
Same as C++, but it's optional as a new line does it for you.
I think it's good pratice to put it explicitly.
We may need a linter at some point.